setup.py

```diff
@@ -66,12 +65,11 @@ def run_tests(self):
 # Extract name and e-mail ("Firstname Lastname <mail@example.org>")
 AUTHOR, EMAIL = re.match(r'(.*) <(.*)>', AUTHOR_EMAIL).groups()
 
-# For some versions of python (2.6) we need the importlib
-# as it is not in the standard install
-try:
-    import importlib
-except ImportError:
-    REQUIREMENTS.append('importlib')
+REQUIREMENTS = ['requests', 'dicttoxml']
```

Comment:
This is left over from when I used pysimplesoap (which could not be read from requirements.txt because it had to be installed from git). I will fix this.  [But in fact, we are doing this the wrong way anyway. The requirements should be listed in setup.py, and pip will get them from there.  It is requirements.txt that needs editing - see https://caremad.io/2013/07/setup-vs-requirement/.  But that is for another PR!